src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
 
-  --editconf           Open the default configuration file in a text editor.
+  --editconf               Open the default configuration file in a text editor.
 
-  -c <command>         Run the specified DOS command before running FILE.
-                       Multiple commands can be specified.
+  --eraseconf              Delete the configuration file.
 
-  -noautoexec          Don't perform any [autoexec] actions.
+  --erasemapper            Delete the mapper file.
```

Comment:
I assume the one defined in the default config? Or whatever `mapperfile` points to after loading the config? Can we make this clearer, I'd be afraid to use it for the first time going by this description.

Reply:
This one is also the default `const auto path = (get_platform_config_dir() / MAPPERFILE).string();`

Reply:
Specified "default"